### PR TITLE
Feature/update pmid prefix

### PR DESCRIPTION
**Pull request by auumgn**

### CODE REVIEW

**Comment by auumgn** [comment]
[PubMed ID resolution prefix](https://trello.com/c/ihObl4nm/7859-pubmed-id-resolution-prefix)

File: orcid-persistence/src/main/resources/db/updates/identifier-types/add-primary-use-and-urls.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		<sql>UPDATE identifier_type SET id_resolution_prefix = 'http://openlibrary.org/b/' WHERE id_name = 'OL';</sql> 
 		<sql>UPDATE identifier_type SET id_resolution_prefix = 'http://www.osti.gov/energycitations/product.biblio.jsp?osti_id=' WHERE id_name = 'OSTI';</sql> 
 		<sql>UPDATE identifier_type SET id_resolution_prefix = 'http://europepmc.org/articles/' WHERE id_name = 'PMC';</sql> 
-		<sql>UPDATE identifier_type SET id_resolution_prefix = 'http://www.ncbi.nlm.nih.gov/pubmed/' WHERE id_name = 'PMID';</sql> 
+		<sql>UPDATE identifier_type SET id_resolution_prefix = 'http://pubmed.ncbi.nlm.nih.gov/' WHERE id_name = 'PMID';</sql>
```
Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/identifier-types/add-primary-use-and-urls.xml:33]: @auumgn this is wrong.

File: orcid-persistence/src/main/resources/db/updates/identifier-types/update-to-https.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 			<sql>UPDATE identifier_type SET id_resolution_prefix = 'https://www.worldcat.org/isbn/' WHERE id_name = 'ISBN';
 			UPDATE identifier_type SET id_resolution_prefix = 'https://arxiv.org/abs/' WHERE id_name = 'ARXIV';
 			UPDATE identifier_type SET id_resolution_prefix = 'https://europepmc.org/articles/' WHERE id_name = 'PMC';
-			UPDATE identifier_type SET id_resolution_prefix = 'https://www.ncbi.nlm.nih.gov/pubmed/' WHERE id_name = 'PMID';
+			UPDATE identifier_type SET id_resolution_prefix = 'https://pubmed.ncbi.nlm.nih.gov/' WHERE id_name = 'PMID';
```
Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/identifier-types/update-to-https.xml:11]: @auumgn this is wrong.

**Review by amontenegro — CHANGES_REQUESTED** [review]

**Comment by amontenegro** [comment]
@auumgn you should create a new liquibase change set (new xml) for doing this.

Please rollback the changes you did in the existing xml files and create a new one.

Thanks

**Review by amontenegro — CHANGES_REQUESTED** [review]
@auumgn you will need to add update-pmid-resolution-prefix.xml  to the db-master.xml file, so, liquibase is aware of it.

**Pull request merged**